Dockerfile

```diff
@@ -0,0 +1,164 @@
+FROM ubuntu:xenial
+MAINTAINER Susanna Kiwala <ssiebert@wustl.edu>
+
+LABEL \
+    description="Image for tools used in the CLE"
+
+RUN apt-get update -y
```

Comment:
Should we combine this `apt-get update` with the following `apt-get install` (or do a second one?)?

I suppose the "efficient" thing to do would be to combine all our `apt-get install`s in this file and combine them into one.  But this is certainly easier for development!